src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -127,6 +127,7 @@ namespace RSS
         QStringList previouslyMatchedEpisodes;
 
         mutable QString lastComputedEpisode;
+        mutable QString lastFullComputedEpisode;
```

Comment:
Please make previous variable to be of type **QStringList** instead of creating this one. And rename to "lastComputedEpisode**s**".